security/pfSense-pkg-sudo/files/usr/local/pkg/sudo.inc

```diff
@@ -203,4 +206,73 @@ function sudo_validate_commands(&$input_errors) {
 		}
 	}
 }
+
+function sudo_sync_config() {
+	global $config;
+	if (is_array($config['hasync'])) {
```

Comment:
You can reverse this check and just return when it's false, then next block will be one level less indented